salt/docker/init.sls

```diff
@@ -15,16 +15,15 @@ docker:
     - require:
       - pkg: docker
 
-{% set docker_opt = "--iptables=false" %}
 {% set docker_reg = pillar.get('docker_registry_mirror', '') %}
```

Comment:
As you are already into this, maybe you could also put `docker_registry_mirror` under `docker` and have 

```
docker:
    # Extra arguments to be passed to the Docker daemon.
    args: '--iptables=false'
    registry_mirror: ''
```